app/config/findbugs.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+
+<!--
+FindBugs exclude filter
+Severity: HARD
+-->
+<FindBugsFilter>
```

Comment:
That was other issue. It was related to lombok. FindBugs is not picking data binding files. Or else, it would definitely show errors. This may be because I have used a custom plugin to load it. Maybe it excludes them automatically. Besides  have removed DataBinding files from checkstyle which was actually checking them. I mean to say I have checked it. DataBinding files are not getting checked